openstack_hypervisor/pci.py

```diff
@@ -106,6 +106,15 @@ def is_network_device(pci_class: str) -> bool:
     return pci_class.startswith("0x02")
 
 
+def is_display_device(pci_class: str) -> bool:
```

Comment:
this function needs a test, not because the complexity of the implementation, but to lock down the behavior.

Reply:
Modified